classes/01class/exercises/c01-git03/mfreitassm/README.md

```diff
@@ -0,0 +1,25 @@
+# c01-git03
+
+## Questions (answer inline)
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+
+Yes, before added the big file (100M) repo size was 168k and after added was 201M.
+
+2. What is the reason for this problem?
+
+A file copy was stored on the folder /.git/objects as history.
+
+3. How do you remove something from Git history after it is pushed to the remote repository? Which commands would you use? 
+
+Git history will need to be rewrited.
+git filter-branch --index-filter "git rm -rf --cached --ignore-unmatch path_to_file" HEAD 
```

Comment:
In a few words, what's it doing?

Reply:
Hi Denis, I am sorry about the delay!, 
I had been off  for the last 2 weeks due to some personal issues .

To answer:
Git filter-branch will do is for each commit in the specified branch, apply filters to the snapshot, and create a new commit. The new commit’s parent will be the filtered version of the old commit’s parent. So it creates a parallel commit DAG.

Reply:
Can you update your answers with that?